File: Python 3/Conditionals/leapYear.py
year = int(input(print("Enter the year:- ")))

if year%4==0:
    if year%400 == 0:
        print(f"{year} is a century year and a leap year"}
    elif year%400 != 0 and year%100 == 0:
        print(f"{year} is a century year but, not a leap year")
    elif year%100 != 0:
        print(f"{year} is not a century year but, a leap year")
else:
    print(f"{year} is not a leap year")

File: Javascript/Conditionals/leap_year.js
const readline = require('readline');

// Create an interface for input and output
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

// Ask the user for the year
rl.question('Enter the year: ', (input1) => {
    var year = parseFloat(input1);
    
    // Compare the years and display the result
    if (year%100 == 0){ 
        if (year%400 == 0)
            console.log("\nResult:- " + year + " is a leap year as well as a century year.\n");
        else
            console.log("\nResult:- " + year + " is not a leap year rather a century year.\n");        
    }
    else{
        if (year%4 == 0)
            console.log("\nResult:- " + year + " is a leap year.\n");
        else
            console.log("\nResult:- " + year + " is not a leap.\n");
    }
    // Close the readline interface
    rl.close();
});

